src/index.js

```diff
@@ -20,19 +36,121 @@ const displayDataTable = (image, title) => {
     <h3>${title}</h3>
   </div>
   <div class="like-comment">
-  <a class="love"><i class="far fa-heart"></i><small>0 likes</small></a>
-  <a class="comment"><i class="far fa-comment"></i><p id="text">comment</p></a>
+  <a class="love" id="${index}like"><i class="far fa-heart"></i><small>0 likes</small></a>
+  <a class="comment" id="${index}"><i class="far fa-comment"></i><p id="text">comment</p></a>
    </div>
    </div>`;
   displayData.appendChild(cardArt);
 };
 
-const displayFromApi = () => {
-  getData(generateUrl)
+function countElements(elem) {
+  return elem.childElementCount;
+}
+
+const closeDisplayPop = (target) => {
+  target.parentElement.parentElement.remove();
+};
+
+const countItems = () => {
+  const itemCount = document.querySelector('#item-count');
+  const mySection = document.querySelector('body > main > section');
+  itemCount.firstChild.innerHTML = `Exhibition ${countElements(mySection)}`;
+};
+
+const countComment = () => {
+  const commentCount = document.getElementById('comment-container');
+  commentCount.previousElementSibling.innerHTML = `Comments ${countElements(commentCount)}`;
+};
+
+function showComment(user, str) {
+  const ulContainer = document.querySelector('#comment-container');
+  const li = document.createElement('li');
+  li.innerHTML = `${user} : ${str}`;
+  ulContainer.appendChild(li);
+}
+
+function displayComments(id) {
+  const showProper = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/9mAPgvMc6PjOJk4JU1ZU/comments?item_id=${id}`;
+  getData(showProper)
+    .then((data) => data.forEach((elem) => showComment(elem.username, elem.comment)))
+    .then(() => countComment());
+}
+
+function addComment(id, user, str) {
+  const data = {
+    item_id: id,
+    username: user,
+    comment: str,
+  };
+
+  postData(commentLink, data)
     .then((data) => {
-      displayData.innerHTML = '';
-      data.forEach((element) => displayDataTable(element.hdurl, element.title));
+      if (data.status === 201) {
+        showComment(user, str);
+      }
     });
+}
```

Comment:
- [ ] Let's fix this POST comment call. For now, it just returns an empty response

![Screenshot from 2021-12-04 18-05-42](https://user-images.githubusercontent.com/40503704/144718137-2bd63a80-bc27-49f3-b2e3-58d236c7cbc2.png)